modules/gapi/test/infer/gapi_infer_ie_test.cpp

```diff
@@ -1062,6 +1182,234 @@ TEST_F(ROIListNV12, Infer2MediaInputNV12)
     validate();
 }
 
+TEST_F(SingleROI, GenericInfer)
+{
+    // Configure & run G-API
+    cv::GMat in;
+    cv::GOpaque<cv::Rect> roi;
+    cv::GInferInputs inputs;
+    inputs["data"] = in;
```

Comment:
BTW can it be constructed like this?
```c++
auto inputs = cv::GInferInputs {
   {"data", in},
   ...
};
```
? This can be done separately, of course.

Reply:
I believe there is no difficulty to do this